backend/tests/integration/assign/test_views.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import json
+
+from helpers.utils import create_record
+from inspire_dojson.utils import get_recid_from_ref
+
+from inspirehep.records.api import AuthorsRecord, LiteratureRecord
+
+
+def test_assign_from_an_author_to_another(inspire_app):
```

Comment:
I see some missing tests in the happy flow:
* `to_author` who doesn't have`stub=true`

I know this test is covering most of it, but adding it will test also the other case.